packages/appstore/src/components/fiat-transaction-list/fiat-transaction-list.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { Statement } from '@deriv/api-types';
+import { getCurrencyDisplayCode, isCryptocurrency, isMobile } from '@deriv/shared';
+import { Text } from '@deriv/components';
+import { useGroupedFiatTransactions } from '@deriv/hooks';
+import { useStore } from '@deriv/stores';
+import { localize } from '@deriv/translations';
+import { getWalletCurrencyIcon } from 'Constants/utils';
+import FiatTransactionListItem from './fiat-transaction-list-item';
+
+const FiatTransactionList = () => {
+    const store = useStore();
+    const {
+        client: { accounts, currency: wallet_currency, loginid },
+        traders_hub: { is_demo },
+        ui: { is_dark_mode_on },
+    } = store;
+    const grouped_transactions = useGroupedFiatTransactions();
+    const linked_accounts = Object.values(accounts)
```

Comment:
For all client accounts from stores also collecting linked accounts, since those might be for app accounts and they have platform

<img width="833" alt="Screenshot 2023-05-31 at 01 41 01" src="https://github.com/binary-com/deriv-app/assets/119863957/04cdfc49-1c37-4f46-8a0c-589b2d243f01">